sdk/python/kfp/dsl/tasks_group.py

```diff
@@ -22,6 +22,7 @@
 from kfp.dsl import pipeline_channel
 from kfp.dsl import pipeline_context
 from kfp.dsl import pipeline_task
+from kfp.dsl.types import type_annotations
```

Comment:
Unused import (applies to the `type_annotations` import throughout)

Reply:
Done.